contrib/docker/README

```diff
@@ -0,0 +1,35 @@
+Avocado-vt inside the container
+
+That a simple way to use avocado-vt in the system when you are worried
+about breaking the system or you wish to test a new version of the
+framework.
+
+How to use avocado-vt inside the container:
+First, you need clone the dockerfile Fedora.dockerfile
+The Fedora-30 uses podman as a default container engine instead of docker
+
+Build an image:
+
+# podman build --build-arg avocado_version=74.0 -t avocado-vt -f Fedora.dockerfile .
+
+To change avocado_version or Fedora distro use next args:
+            --build-arg avocado_version=74.0
```

Comment:
Like I said elsewhere, any reason for not using ALL CAPS here too? I mean, `AVOCADO_VERSION` ?

Reply:
Cleber, thanks for the review.

Reply:
@clebergnu 
Sorry Cleber, you are totally right
I've used our local variable from jobs, it's just one reason :(
And when I published file I provide mistake with incorrect variables name and so on